cdap-ui/app/hydrator/services/detail/actions/pipeline-detail-metrics-actions.js

```diff
@@ -67,9 +67,23 @@ angular.module(PKG.name + '.feature.hydrator')
              *  Since the parent block is already a poll, we don't need another poll for
              *  the values of each metrics.
              **/
+            var payload = {
+              metric: {
+                context: params,
+                names: metricQuery,
+                startTime: '0',
+                endTime: 'now',
+                resolution: '2147483647s'
```

Comment:
It's the time interval for each set of data point for a metric.
2147483647s is basically 60 years or essentially forever, it's set to that to mimic and get the exact same response from the server when we make the api call with the params in the url.

note: When we make the api call with the params in the url, 'startTime', 'endTime', and 'resolution' are not sent in, they're only sent in when making the api call with the params in the body.